Add Keras BOW model example

sararob opened the pull request:
@random-forests could you review? Added the latest code from our blog post + BQ query. Should I also add the CSV?

random-forests commented:
Thanks! This is great. Just one quick improvement. In cell #6 of the notebook, our code shows loading the CSV from disk. To make it easier for folks (so they don't have to run the query to generate the data), can we also upload the CSV file to a GS bucket (and leave a comment with the URL in that cell)?

random-forests commented:
Oh yes! At the bottom of the Cell 2 (that imports everything), could please add a comment that says something like "This code was tested with TensorFlow Version v1.3." and after that, could you print out the user's version, like this? "print('You have:', tf.__version__)" Just in case the imports change a bit in v1.4.

sararob commented:
Updated! We didn't have a `import tensorflow as tf` import since we're using Keras so I added one. Also made a public GCS file and linked to it in the comment.
